### PR TITLE
#2053 Pass fieldsettings to region field collection

### PR DESCRIPTION
**Pull request by sit-harry**
Small change to ContentTypeBuilder and ContentTypeService to pass field settings to the field if it is a single field region

### CODE REVIEW

**Comment by sit-harry** [comment]
@dotnet-policy-service agree company="Sound in Theory"

File: core/Piranha.Manager/Services/ContentTypeService.cs [hunk 1]
```diff
@@ -267,6 +267,7 @@ private async Task<RegionItemModel> CreateRegionAsync(ContentTypeBase type, stri
             {
                 field.Model = (IField)regionModel;
                 field.Meta.NotifyChange = true;
+                field.Meta.Settings = fieldType.Settings;
```
Comment (tidyui) [inline core/Piranha.Manager/Services/ContentTypeService.cs:270]: Isn't this already handled at line 255? My guess is that you solved the underlying problem in the `ContentTypeBuilder`.

Comment (sit-harry) [reply core/Piranha.Manager/Services/ContentTypeService.cs]: Oops, yes you're right! I was tinkering on an older version of Piranha that didn't have the assignment to settings and must have overlooked this. 

I have removed this line in the latest commit.

**Comment by tidyui** [comment]
Fixes #2053

**Pull request merged**